aws/aws.go

```diff
@@ -374,7 +373,25 @@ func GetAllResources(targetRegions []string, excludeAfter time.Time, resourceTyp
 				resourcesInRegion.Resources = append(resourcesInRegion.Resources, dbInstances)
 			}
 		}
-		// End RDS DB resources
+		// End RDS DB Instances
+
+		// RDS DB Clusters
+		// These reference the Aurora Clusters, for the use it's the same resource (rds), but AWS
+		// has different abstractions for each.
+		dbClusters := DBInstances{}
```

Comment:
I think you need a different struct here so that the right nuking function gets called.

Reply:
Wops, it was calling the `nukeDBInstances` thanks!